File: www_root/src/Model/Entity/Agent.php
<?php
namespace App\Model\Entity;

use Cake\ORM\Entity;

/**
 * Agent Entity
 *
 * @property int $id
 * @property int $character_id
 * @property int $ships_destroyed
 * @property int $isk_destroyed
 *
 * @property \App\Model\Entity\Character $character
 */
class Agent extends Entity
{

    /**
     * Fields that can be mass assigned using newEntity() or patchEntity().
     *
     * Note that when '*' is set to true, this allows all unspecified fields to
     * be mass assigned. For security purposes, it is advised to set '*' to false
     * (or remove it), and explicitly make individual fields accessible as needed.
     *
     * @var array
     */
    protected $_accessible = [
        '*' => true,
        'id' => false
    ];
}

File: web/rules/models/agent.py
class Agent(object):
    def __init__(
        self,
        character_id,
        character_name,
        ships_destroyed,
        isk_destroyed,
        position = 0):
        self.character_id = character_id
        self.character_name = character_name
        self.ships_destroyed = ships_destroyed
        self.isk_destroyed = isk_destroyed
        self.position = position
